### PR TITLE
Added repo size conversion to github repo cards.

### LINKED ISSUE #525: Add repo size conversion to github repo cards
Cards currently only show repo sizes in kb. This looks ugly when a repo size is in mb or greater.

We should add size conversion and format the output appropriately

### PR DESCRIPTION
**Pull request by Vinitvh**
Added repo size conversion to GitHub repo cards closes #525

### CODE REVIEW

File: src/components/githubRepoCard/GithubRepoCard.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import "./GithubRepoCard.scss";
 import {Fade} from "react-reveal";
+import {format} from "./sizeConversion";
```
Comment (kartikcho) [inline src/components/githubRepoCard/GithubRepoCard.js:4]: Can we rename this to something descriptive like `formatFileSizeDisplay`?

File: src/components/githubRepoCard/sizeConversion.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const format = value => {
```
Comment (kartikcho) [inline src/components/githubRepoCard/sizeConversion.js:1]: You could move this function into a `utils.js` file in `/src`. That way we can add more functions in one place or share it amongst files if needed

**Review by kartikcho — COMMENTED** [review]
Need to address some changes and then we're good to merge this!

File: src/components/githubRepoCard/sizeConversion.js (hunk 2 above)
Comment (Vinitvh) [reply src/components/githubRepoCard/sizeConversion.js:1]: Okay will make necessary changes.

**Review by kartikcho — APPROVED** [review]
Great PR! Thanks

**Comment by kartikcho** [comment]
@allcontributors please add @Vinitvh for code

**Pull request merged**